src/source/PeerConnection/SessionDescription.h

```diff
@@ -44,6 +45,7 @@ extern "C" {
 #define DEFAULT_PAYLOAD_MULAW   (UINT64) 0
 #define DEFAULT_PAYLOAD_ALAW    (UINT64) 8
 #define DEFAULT_PAYLOAD_OPUS    (UINT64) 111
+#define DEFAULT_PAYLOAD_AAC     (UINT64) 96
```

Comment:
We should probably remove these hardcoded payload types altogether: https://issues.webrtc.org/issues/42231779
Not necessary for part of this PR.

Reply:
We're using it as a part of `setPayloadTypesForOffer` actually